src/modules/user/user.service.ts

```diff
@@ -0,0 +1,41 @@
+import {Injectable} from '@nestjs/common';
+import {getRepository} from 'typeorm';
+import {CreateUserDto} from './dto/create-user.dto';
+import {User} from './user.entity';
+
+@Injectable()
+export class UserService {
+  createUser(createUserDto: CreateUserDto): Promise<User> {
+    const user = new User();
+    user.id = createUserDto.id;
+    user.password = createUserDto.password;
+    user.nickname = createUserDto.nickname;
+    user.email = createUserDto.email;
+    user.birthday = createUserDto.birthday;
```

Comment:
createUser DTO 파일이 존재해서 중복되는 것 같은데 respository만 inject하는게 어떨까요?
+ 기존 유저 존재하는 지에 대한 로직 처리도 있으면 좋을 것 같습니다.
```suggestion
  createUser(createUserDto: CreateUserDto) {
    const userData = await this.userRepository.save({
        email: createUserDto.email,
        name: createUserDto.name,
        ....작성
        password: createUserDto.password,
      });
  
      delete userData.password;
  
      return userData;
  }
```